### DISCUSSION TITLE
How to skip the rest of the steps

### DISCUSSION THREAD

**Discussion by kbbgl** [community/community#46330] [Actions]
Let's say I have 3 steps in a job. I want to prevent the last 2 from running if a certain condition is not met. 

Do I need to put the same `if` condition in both of the last 2 steps or only on the 2nd?

**Comment by SilasRodrigues19** [comment]
Hey @kbbgl 

You can use `return` at the end of the if statement, if the condition is satisfied it will stop the execution there

**Reply by kbbgl** [reply]
I'll try it out. Thanks @SilasRodrigues19!

**Reply by kgal-pan** [reply]
Hey @SilasRodrigues19 - I attempted this change [here](https://github.com/demisto/content/pull/24228/commits/69549969637a7222617a1ca43fa5cf4393a5094a#diff-04f7f7bfba93cb5e3702478fe2749252be924ea8c54ea9a23d47aa5e5557830eR33) but it resulted in [an invalid workflow](https://github.com/demisto/content/actions/runs/4096207560):

```
[Invalid workflow file: .github/workflows/review-release-notes.yml#L33](https://github.com/demisto/content/actions/runs/4096207560/workflow)
The workflow is not valid. .github/workflows/review-release-notes.yml (Line: 33, Col: 13): Unexpected symbol: 'return'. Located at position 109 within expression: steps.changed-files.outputs.all_changed_files == '' || steps.changed-files.outputs.all_changed_files == ' ' return
```

Please advise.

**Reply by kgal-pan** [reply]
Looks like there's no real way to do this as it's been broken since V1, see https://github.com/actions/runner/issues/662.
